modules/instrument_manager/php/instrument_manager.class.inc

```diff
@@ -190,8 +205,6 @@ class Instrument_Manager extends \NDB_Menu_Filter
                         json_encode(
                             array('message' => self::UPLOAD_NO_INSTALL)
                         )
-                    )
```

Comment:
don't think these extra round brackets need to be removed

Reply:
on testing, error_log `PHP Parse error:  syntax error, unexpected '}', expecting ',' or ')' in /var/www/loris/modules/instrument_manager/php/instrument_manager.class.inc on line 208`

Reply:
Hm okay I think this was a rebase error. i'll fix this.